hera_qm/utils.py

```diff
@@ -183,10 +183,10 @@ def get_metrics_ArgumentParser(method_name):
         a.add_argument('--time_threshold', default=0.05, type=float,
                        help='Fraction of times required to trigger broadcast across'
                        ' time (single frequency). Default is 0.05.')
-        a.add_argument('--ex_ants', default='', type=str,
+        a.add_argument('--ex_ants', default=None, type=str,
                        help='Comma-separated list of antennas to exclude. Flags of visibilities '
                        'formed with these antennas will be set to True.')
-        a.add_argument('--metrics_json', default='', type=str,
+        a.add_argument('--metrics_json', default=None, type=str,
```

Comment:
yeah, I was torn about this. I thought we'd want to keep the old naming because this is a legacy wrapper as it is. But I'm happy to change it if I'm the minority on this.

Reply:
could always overload the destination with both inputs. have lines like:
``` 
a.add_argument('--metrics_file', dest='metrics_file', type=str, action='append')
a.add_argument('--metrics_json', dest='metrics_file', type=str, action='append')
a.set_defaults(metrics_file=None)
```
that way someone could specify either or both of the inputs. The draw back is that we could end up with something like this:
```
In [17]: a.parse_args('--metrics_file=foobar --metrics_json=foobar'.split())
Out[17]: Namespace(metrics_file=['foobar', 'foobar'])
```
but if we're not worried about getting the same argument multiple times it doesn't matter.  Especially since this is for `process_ex_ants`, it won't append the same antenna multiple times and if you loop over a file twice it won't take long and it was your fault anyway.